backend/core/src/shared/password-regex.ts

```diff
@@ -0,0 +1 @@
+export const passwordRegex = /^(?=.*?[A-Z])(?=.*?[a-z])(?=.*?[0-9]).{8,64}$/
```

Comment:
I noticed, on the frontend - the helper message does not contain any information about at least one capital letter.

Reply:
Great find, I'll ask Jesse/Kathy which one should change

Reply:
Looks like we want to lower the regex requirements and keep the copy, I updated the regex 👍